test/functional-api/integration/aspect/aspect-templates.it-spec.ts

```diff
@@ -0,0 +1,600 @@
+    });
+    test('Create aspect with existing aspect template, and remove the aspect template, doesnt change the aspect type', async () => {
+      // Act
+      await deleteAspectTemplate(aspectTemplateId);
+
+      const getAspectsData = await aspectDataPerContext(
+        0,
+        entitiesId.hubId,
+        entitiesId.challengeId,
+        entitiesId.opportunityId
+      );
+      const data = getAspectsData[0];
+
+      // Assert
+      expect(aspectTypeFromHubTemplate).toEqual(templateType);
+      expect(data).toEqual(aspectDataCreate);
+    });
+  });
+});
+
+describe('Aspect templates - CRUD Authorization', () => {
+  const templateType = 'testTemplateType';
+  beforeAll(async () => {
+    await assignUsersToHubAndOrg();
+  });
+  describe('Aspect templates - Create', () => {
+    describe('DDT user privileges to create hub aspect template - positive', () => {
+      // Arrange
+      afterEach(async () => {
+        await deleteAspectTemplate(aspectTemplateId);
+      });
+      test.each`
+        userRole                 | templateTypes | message
+        ${TestUser.GLOBAL_ADMIN} | ${'GA type'}  | ${'"data":{"createAspectTemplate"'}
+        ${TestUser.HUB_ADMIN}    | ${'HA type'}  | ${'"data":{"createAspectTemplate"'}
+      `(
+        'User: "$userRole" get message: "$message", whe intend to create hub aspect template ',
+        async ({ userRole, templateTypes, message }) => {
+          // Act
+          const resCreateAspectTempl = await createAspectTemplate(
+            entitiesId.hubTemplateId,
+            templateTypes,
+            'test default description',
+            'test title',
+            'test description',
+            ['tag11', 'tag12'],
+            userRole
+          );
+          aspectTemplateId =
+            resCreateAspectTempl.body.data.createAspectTemplate.id;
+
+          // Assert
+          expect(resCreateAspectTempl.text).toContain(message);
+        }
+      );
+    });
+
+    describe('DDT user privileges to create hub aspect template - negative', () => {
+      // Arrange
+      test.each`
+        userRole                   | message
+        ${TestUser.HUB_MEMBER}     | ${'Authorization: unable to grant \'create\' privilege: templates set create aspect template:'}
+        ${TestUser.NON_HUB_MEMBER} | ${'Authorization: unable to grant \'create\' privilege: templates set create aspect template:'}
+      `(
+        'User: "$userRole" get message: "$message", whe intend to create hub aspect template ',
+        async ({ userRole, message }) => {
+          // Act
+          const resCreateAspectTempl = await createAspectTemplate(
+            entitiesId.hubTemplateId,
+            templateType,
+            'test default description',
+            'test title',
+            'test description',
+            ['tag11', 'tag12'],
+            userRole
+          );
+
+          // Assert
+          expect(resCreateAspectTempl.text).toContain(message);
+        }
+      );
+    });
+  });
```

Comment:
Merge with previouse DDT block.

Reply:
I've split them the DDT blocks for `createAspectTemplate` intentionally, as the first suites, where the entity is created successfully, I am gathering the identifier in order to delete it after the row pass. In the second block, an entity is not created, hence, the test fails, before reaching the assertion block.